Makefile

```diff
@@ -1,4 +1,5 @@
-all:
+all: apidocs install
```

Comment:
As I already wrote in my last comment:

I don't like the idea that a "make all" will trigger installation of files in the filesystem. "make all" is mostly done to compile software or e.g. in perl packages it`s used to prepare the distribution of the package (copying to ./blib). Nobody would expect an "make all" to install files in the filesystem. Here we should be consistent to established Unix/Linux standards